package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "camera-controls",
-  "version": "2.2.1",
+  "version": "0.0.0-semantic-release",
   "author": "Yomotsu",
   "license": "MIT",
   "main": "dist/camera-controls.js",
   "jsnext:main": "dist/camera-controls.module.js",
   "module": "dist/camera-controls.module.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
```

Comment:
specifying what files to include in the package (`package.json` and `readme.md` will be automatically added, no need to specify them)